core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -670,9 +670,12 @@ private[spark] class TaskSetManager(
           }
           if (blacklistedEverywhere) {
             val partition = tasks(indexInTaskSet).partitionId
-            abort(s"Aborting $taskSet because task $indexInTaskSet (partition $partition) " +
-              s"cannot run anywhere due to node and executor blacklist.  Blacklisting behavior " +
-              s"can be configured via spark.blacklist.*.")
+            abort(s"""
+              |Aborting $taskSet because task $indexInTaskSet (partition $partition)
+              |cannot run anywhere due to node and executor blacklist.
+              |Most recent failure:
+              |${taskSetBlacklist.getLatestFailureReason}\n
```

Comment:
You don't have to add "\n", since you already use triple quoted format. Please trying to get understand the basic of this API before modifying it. 

Also verified in locally before pushing a new commit, and update the PR description to reflect your new format accordingly.

Reply:
Actually i tested locally with something like below:
```
scala> val s = s"""
     | sss\n
     | sss"""
scala> print(s)

sss

sss
```
And i found that there is no need to add \n since triple quoted designed to avoid such character.Sorry for that,and i have updated before your newest comment .Thanks @jerryshao

Reply:
Are you sure?

```scala
scala> val s = """fdsafdsa\n\nfdsafdsa"""
s: String = fdsafdsa\n\nfdsafdsa
```

Reply:
NVM, looks like whether using string interpolation the result is different.

Reply:
It is my fault,scala version affected.My default scala is 2.10.4.And below is the result i tested with scala 2.11.2:
```
scala> val s="""ss\nss
     | sss\n"""
s: String =
ss\nss
sss\n

scala> print(s)
ss\nss
sss\n
```